eng/Baseline.Designer.props

```diff
@@ -2,146 +2,146 @@
 <Project>
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-    <AspNetCoreBaselineVersion>3.1.0</AspNetCoreBaselineVersion>
+    <AspNetCoreBaselineVersion>3.1.2</AspNetCoreBaselineVersion>
```

Comment:
This looks wrong because it doesn't align with Baseline.xml

Reply:
Fixed